swift/ql/lib/codeql/swift/security/CleartextLoggingQuery.qll

```diff
@@ -25,6 +25,8 @@ module CleartextLoggingConfig implements DataFlow::ConfigSig {
   predicate isAdditionalFlowStep(DataFlow::Node n1, DataFlow::Node n2) {
     any(CleartextLoggingAdditionalFlowStep s).step(n1, n2)
   }
+
+  predicate observeDiffInformedIncrementalMode() { any() }
```

Comment:
What does this mean?

(I wrote many of these queries, I'm sure I can figure out if it's appropriate)

Reply:
This line enables diff-informed mode, i.e. only showing results that are in a diff range. There is currently some internal documentation [here](https://github.com/github/codeql-core/blob/main/wiki/diff-informed-queries.md), not sure if it's documented publicly yet.